test/unit/unit-test.spec.js

```diff
@@ -11,15 +11,14 @@
 // see: https://github.com/digitalbazaar/bedrock-test/issues/16 . c8 is drop replacement for nyc coverage reporting tool
 /*global describe, it*/
 
-import helloWorld from "../../src/index.js";
-import * as assert from 'assert';
-import * as chai from 'chai';
+const assert = require('assert');
```

Comment:
Would be good to add unit tests from the beginning as this could be a widely reused lib. 
Some examples here: https://github.com/aicore/Core-Analytics-Server/tree/main/test/unit